argilla-server/docker/argilla-hf-spaces/Dockerfile

```diff
@@ -44,7 +44,9 @@ RUN \
     apt-get remove -y wget gnupg && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
-    rm -rf /packages
+    rm -rf /packages && \
+    # Install pwgen curl and jq
+    apt-get update && apt-get install -y curl jq pwgen
```

Comment:
@frascuchon  why we added this?

Reply:
Go it. It's used to get the default user from the API (using curl and jq) and generate a new password.

Reply:
I'm gonna moved it to install them after the first `apt-get update`

Reply:
Done!